ConnectorRhino/ConnectorRhino/ConnectorRhinoShared/UI/ConnectorBindingsRhino.cs

```diff
@@ -602,6 +604,7 @@ public override async Task<StreamState> ReceiveStream(StreamState state, Progres
 
             layers.Add(layer.FullPath, layer);
           }
+          RhinoDoc.LayerTableEvent += RhinoDoc_LayerChange;
           #endregion
```

Comment:
Just wondering if we should delay adding this event even further down the method to prevent possible mishaps if a user modifies a layer while baking?